src/scripts/main.js

```diff
@@ -1,7 +1,36 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const letBody = document.querySelector('body');
```

Comment:
```suggestion
  const letBody = document.body;
```
До `body` можна звертатись простіше)
Підкажи, будь ласка, чому вибрав таку назву для змінної? Чому не просто `body`?

Reply:
название выбирал, чтобы оно не пересекалось с body
но раз можно обратиться проще к body, то в моем названии и смысла нет